src/sentry/static/sentry/app/components/noProjectMessage.jsx

```diff
@@ -63,13 +76,22 @@ export default class EmptyState extends React.Component {
   }
 }
 
+const StyledPageHeading = styled(PageHeading)`
+  font-size: 28px;
```

Comment:
Unfortunately it is a little larger than`fontSizeLarge`, which is our largest size (22px). I did try that size, and it felt too small. 

We could add a new thing called like `headlineSize` or something, but headlines are usually so rare in the product, I doubt it would get much use.